flink-metrics/flink-metrics-slf4j/src/test/java/org/apache/flink/metrics/slf4j/Slf4jReporterTest.java

```diff
@@ -82,17 +77,17 @@ public void testAddCounter() throws Exception {
                         + ": 0";
 
         reporter.report();
-        assertThat(
-                testLoggerResource.getMessages(), hasItem(containsString(expectedCounterReport)));
+        assertThat(testLoggerResource.getMessages())
+                .anyMatch(logOutput -> logOutput.contains(expectedCounterReport));
     }
 
     @Test
-    public void testAddGauge() throws Exception {
+    void testAddGauge() throws Exception {
         String gaugeName = "gauge";
 
         Gauge<Long> gauge = () -> null;
         reporter.notifyOfAddedMetric(gauge, gaugeName, metricGroup);
-        assertTrue(reporter.getGauges().containsKey(gauge));
+        assertThat(reporter.getGauges().containsKey(gauge));
```

Comment:
```suggestion
        assertThat(reporter.getGauges()).containsKey(gauge);
```
Misplaced parenthesis causes this test to be ignored.